osm2streets/src/lanes/mod.rs

```diff
@@ -355,3 +356,73 @@ impl fmt::Display for Direction {
         }
     }
 }
+
+#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq, PartialOrd, Ord, Serialize, Deserialize)]
```

Comment:
nitpick: I'm guessing you copied the `derive`. Do we need to hash, serialize, order these?